packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -4,48 +4,69 @@ import { Button } from './buttons/Button';
 import { ScrollOnboardingVersion } from '../lib/featureValues';
 import { cloudinary } from '../lib/image';
 import AnalyticsContext from '../contexts/AnalyticsContext';
-import { AnalyticsEvent, TargetType } from '../lib/analytics';
+import { AnalyticsEvent } from '../lib/analytics';
 
-const GaussianBlur = (): ReactElement => {
-  return (
-    <div className="absolute w-52 h-52 rounded-full opacity-32 blur-2xl bg-theme-color-cabbage" />
-  );
-};
+type ScrollOnboardingVersionMap = Partial<
+  Record<ScrollOnboardingVersion, string>
+>;
```

Comment:
I think we can remove the partial.

```suggestion
type ScrollOnboardingVersionMap =  Record<ScrollOnboardingVersion, string>;
```

Reply:
Added the `Partial` because of `ScrollOnboardingVersion.Control`, did not want to add `''` mappings to each map object. Related to this thread https://github.com/dailydotdev/apps/pull/1629#discussion_r1113918285 `classNames` will take care of `undefined` below.

Reply:
Good point. Thought of making it consistent since on line 28 we had an empty string too. But this is nothing blocking, this comment should be resolved! 💯